internal/icinga2/api_responses.go

```diff
@@ -176,40 +181,29 @@ type StateChange struct {
 	CheckResult     CheckResult `json:"check_result"`
 	DowntimeDepth   int         `json:"downtime_depth"`
 	Acknowledgement bool        `json:"acknowledgement"`
+	IsFlapping      bool        `json:"-"` // TODO: Need a workaround for this!! Sending extra API query is ridiculous.
```

Comment:
I'm quite unsure about this one as the Icinga 2 event streams for `StateChange` don't contain that bit information, so I'm just clueless as to how to proceed here. Apart from that and the handling of the `ACK' events, this PR should just work smoothly.